test/standalone/Vegetation/plant_hydraulics_test.jl

```diff
@@ -215,7 +215,7 @@ for FT in (Float32, Float64)
             SAI,
             RAI,
         )
-        K_sat_plant = 1.8e-8 # m/s.
+        K_sat_plant = 1.0 # m/s.
```

Comment:
why did you change this value (and T, LAI further down) to 1?

Reply:
I changed it back - thought it turns out the LAI one was a typo! and our test must not have caught it because it was not strict enough